jobs/forms.py

```diff
@@ -19,3 +19,18 @@ class Meta:
             'title': 'Job title'
         }
 
+
+class MeetupForm(forms.ModelForm):
+
+    class Meta:
+        model = Meetup
+        exclude = ['reviewer', 'review_status', 'reviewers_comment',
+            'ready_to_publish', 'published_date', 'created', 'expiration_date'
+        ]
+
```

Comment:
there is a problem with this approach - if somebody adds a company name which already exists in the database and provides different website, we get integrity error. so I thought if we use dropdown list, it somehow solves it - you can still add your company on the very same site if it doesn't appear in the dropdown. it could be resolved in other ways too.
as far as I saw, django notifies if something is depreciated, and exclude doesn't give any warnings, so I assume it may be used.